### ISSUE TITLE
Can't load mpv

### ISSUE THREAD

**Issue by lucas-04** [URUWorks/SubtitleWorkshop#52]
![image](https://github.com/URUWorks/SubtitleWorkshop/assets/95548091/f0ca4956-0052-40a6-8ff1-cb2ce26d62ae)

**Comment by Krysztophe** [comment]
Worked for me:
```bash
sudo apt install libmpv1
```

**Comment by lucas-04** [comment]
libmpv1 is already the newest version (0.32.0-3).

**Comment by Krysztophe** [comment]
I've got a newer one:
```
 *** 0.34.1-1ubuntu3 500
        500 http://fr.archive.ubuntu.com/ubuntu jammy/universe amd64 Packages
        100 /var/lib/dpkg/status
```

**Comment by lucas-04** [comment]
This version here was compiled on my pc, the precompiled version of URUWorks does not work on debian 11 because of an "outdated" library, I don't know if this has something to do with it. I had difficulties to prepare lazarus (The UWLazarus package was strange) but I did not touch a line of code before compiling.
![image](https://github.com/URUWorks/SubtitleWorkshop/assets/95548091/71f3d6e2-1bf6-430c-a475-bee22cb20453)

**Comment by URUWorks** [comment]
In a few days we will publish a new version.

@Krysztophe,
Is it possible to download a zip file with pre-compiled lib?

@lucas-04, 
Você é um programador e quer colaborar?

**Comment by lucas-04** [comment]
> In a few days we will publish a new version.
> 
> @Krysztophe, Is it possible to download a zip file with pre-compiled lib?
> 
> @lucas-04, Você é um programador e quer colaborar?

Sim, mas eu sou muito inexperiente. Acho que o máximo que eu posso fazer no momento é fornecer uma tradução da interface e relatar issues

**Comment by URUWorks** [comment]
legal, somente Linux? Windows?

**Comment by lucas-04** [comment]
Linux e windows

**Comment by GitGianluc** [comment]
> In a few days we will publish a new version.

@URUWorks can't wait to test the new upcoming release! Hope that bug #47 will be fixed…
